src/PoolManager.sol

```diff
@@ -142,9 +142,13 @@ contract PoolManager is IPoolManager, Fees, NoDelegateCall, ERC6909Claims {
     }
 
     function _accountDelta(Currency currency, int128 delta) internal {
+        _accountDeltaFor(currency, delta, msg.sender);
+    }
+
+    function _accountDeltaFor(Currency currency, int128 delta, address target) internal {
```

Comment:
nit: is `function _accountPoolBalanceDelta` making too many jumps (new lines 166/167)? Maybe it can just take the approach on line 145.

Reply:
other nit, some helpers are above the main function, and some are below. Consistency would be easier to read

Reply:
yeah good idea i'll get rid of the doubling up of accountDelta and accountPoolBalanceDelta and just make every call pass in the target address